Week1/Homework/js-exercises/Exercise5.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+const z = 7.25;
+console.log(z);
+const a = 7;
+console.log(a);
+console.log(Math.max(z, a)); // expected output: 7.25

```

Comment:
The result need to be assigned to a variable, for example `const maxNumber = Math.max(z, a);`